src/components/GenreSelect/__tests__/genreSelect.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import GenreSelect from './../GenreSelect';
+
+describe('GenreSelect', () => {
+	const genres = ['Action', 'Comedy', 'Drama'];
+	const selectedGenre = 'Comedy';
+	const onSelect = jest.fn();
```

Comment:
you can define object with props type:
```ts
const props: GenreSelectProps = {
  genres: ['Action', 'Comedy', 'Drama'],
  selectedGenre: 'Comedy',
  onSelect: jest.fn(),
}
```

and then use just this object in props:

```jsx
<GenreSelect {...props} />
```

or if you need to redefine some prop:

```jsx
<GenreSelect {...props} selectedGenre="Action"} />
```